source/computer/macos-setup.rst

```diff
@@ -274,6 +274,16 @@ Homebrew 将 GNU 实用工具安装在 :file:`/usr/local/bin` 目录下，但在
 在遇到不兼容的情况下，可以考虑使用 GNU 工具（有前缀 ``g``），但在写脚本时，
 要额外注意脚本的可移植性。
 
+`tldr <https://tldr.sh/>`__ 是一个提供命令的常用用法和示例的命令行工具，
```

Comment:
I will submit another PR to list all the commands in macOS.

Reply:
Good.